catt/cli.py

```diff
@@ -176,12 +189,15 @@ def process_subtitle(ctx, param, value):
 @click.option(
     "--no-subs", is_flag=True, default=False, help="Don't try to load subtitles automatically from the local folder."
 )
+@click.option("-y", "--ytdl-option", type=YTDL_OPT, multiple=True, metavar="OPT", help="Youtube-dl option.")
 @click.pass_obj
-def cast(settings, video_url, subtitle, force_default, random_play, no_subs):
+def cast(settings, video_url, subtitle, force_default, random_play, no_subs, ytdl_option):
```

Comment:
Can you rename `ytdl_option` to `ytdl_options` everywhere *except* this function? When the user passes it, it's one option, but when we receive it (i.e. from `setup_cast` onwards), it's going to be a tuple of multiple options, so it would be more accurate to say "options".

Reply:
We also have not mentioned anywhere how pass youtube-dl  parameters, so I have updated the options short help text with *"Youtube-dl option. Should be passed as `key=value` format"*

Reply:
Good call, although you might want to specify `-y key=value` so people don't think it's `--key=value` or something similar.